File: experiments/issue-282/parity/check.php
<?php

// Print what the PHP implementation makes of the document.

declare(strict_types=1);

require dirname(__DIR__, 3) . '/php/vendor/autoload.php';

use LinkFoundation\LinksNotation\Formatter;
use LinkFoundation\LinksNotation\Parser;

$document = file_get_contents(__DIR__ . '/document.lino');
echo Formatter::formatLinks((new Parser())->parse($document)), "\n";

File: experiments/issue-282/parity/check.py
"""Print what the Python implementation makes of the document."""

import sys
from pathlib import Path

sys.path.insert(0, str(Path(__file__).resolve().parents[3] / "python"))

from links_notation import Parser, format_links  # noqa: E402

document = (Path(__file__).parent / "document.lino").read_text(encoding="utf-8")
print(format_links(Parser().parse(document)))
